tests/devices/test_default_qutrit_mixed.py

```diff
@@ -80,9 +1055,7 @@ def test_rng_as_seed(self):
 
 @pytest.mark.jax
 class TestPRNGKeySeed:
-    """Test that the device behaves correctly when provided with a PRNG key and using the JAX interface"""
-
-    # pylint: disable=too-few-public-methods
+    """Test that the device behaves correctly when provided with a JAX PRNG key."""
```

Comment:
We recently updated how `PRNGKey`s are used for execution. These tests might have to be updated accordingly.

Reply:
Would that affect how it is written for this device, do I need to update that?

Reply:
Oops I didnt see above, my bad I will look into it

Reply:
I updated it to be like how `default_qubit` tests are now.

Reply:
The updates were seperate from how QutritMixed sampling happens I tried to make the tests more similar but I have to leave some tests out as it will require changing some of the sampling code which I think should be a separate PR. 

Namely I left out `test_prng_key_multi_tapes`  as it does not use a new PRNG and the `test_get_prng_keys` test as that method has not been added

Reply:
Ok, we can cover this in the future in that case.

Reply:
Sounds good to me. In that case, we don't need to add the `_prng_seed` argument, as that is used for resetting the PRNGKey before new executions.

Reply:
Good poing i will remove